README.md

```diff
@@ -24,7 +24,7 @@ python3 -m pip install -r requirements.txt
 - Edit the variants: `supported_variants` and time controls: `supported_tc` from the config.yml as necessary
 
 ### Windows:
-- Here is a video on how to install the bot: (https://youtu.be/w-aJFk00POQ). Or you may proceed to the next steps.
+- Here is a video on how to install the bot: [click here](https://youtu.be/w-aJFk00POQ). Or you may proceed to the next steps.
```

Comment:
```suggestion md
- Here is [a video on how to install the bot](https://youtu.be/w-aJFk00POQ). Or you may proceed to the next steps.
```

Noooo "click here" is bad for a11y
https://www.google.com/search?client=firefox-b-1-d&q=click+here
First result: https://www.w3.org/QA/Tips/noClickHere

Reply:
Wait why is this resolved, this isn't fixed